notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>Git source could be GitHub (from your fork), or CodeCommit if you don’t supply your access token. Also I'm considering changing this workshop to be a <a href="https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template" target="_blank">GitHub template</a> which would remove the need for "Forking".</p><p> </p><p>You describe the build stage as creating CFN templates which are run later - this is not YET the case, although I plan to do this - so just require some work from me here to be in sync.</p><p> </p><p>I’m not sure (through&nbsp;assets/deploy-model-prod.yml) is the clear here.  The actual template used to deploy prod is transformed to <code>assets/template-model-prod.yml</code> which  uploads zip files to s3 for Lambda Functions  - is this worth mentioning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I'll add the comment on GitHub vs CodeCommit. Since the Github template hasn't been implemented yet, I think it might be better to leave it out. Unless you plan to implement that within the next few days?

Regarding the CFN templates, should I change the description to match the current state, or will you be implementing this change in the next couple of days? 

And good point on the deploy prod template. I'll adjust the description of that.

Reply:
<p>Okay, maybe just remove the reference to "Fork" then, as then it won't change if they have a GitHub Template or Fork.</p>